df-valve/src/main/java/com/hashmapinc/tempus/witsml/valve/dot/DotTranslator.java

```diff
@@ -16,8 +16,12 @@
 package com.hashmapinc.tempus.witsml.valve.dot;
 
 import com.hashmapinc.tempus.WitsmlObjects.AbstractWitsmlObject;
+import com.hashmapinc.tempus.WitsmlObjects.Util.WellConverter;
+import com.hashmapinc.tempus.WitsmlObjects.Util.WellboreConverter;
 import com.hashmapinc.tempus.WitsmlObjects.Util.WitsmlMarshal;
+import com.hashmapinc.tempus.WitsmlObjects.v1311.ObjWellbore;
```

Comment:
Consider NOT importing these objects where both 1311 and 1411 classes with the same name are used. It's verbose to use the full name, but it's much clearer. Here, `ObjWellbore` usage will be 1311 but `ObjWellbores` is 1411.